site/pages/api/get-all-disruptions.test.ts

```diff
@@ -140,6 +141,8 @@ describe("getAllDisruptions", () => {
     });
 
     describe("formatSortedDisruptions", () => {
+        MockDate.set("2023-10-18");
```

Comment:
think it would be good to add mockDate.reset() after this test runs to ensure it doesn't cause issue with other test files, add the below on line 145:
```
afterAll(() => {
        MockDate.reset();
    });
```

Reply:
done